manage.py

```diff
@@ -46,10 +53,24 @@ def test(all=False):
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
 
+    q = Queue('default', connection=conn)
     while True:
         logging.info("Starting scrape...")
-        scrape_and_categorize_articles()
-        logging.info("Finished scrape. Let's run it back...")
+        job = q.enqueue(scrape_and_categorize_articles)
```

Comment:
Instead of making the full scrape one task, it might help to break it down into feed scrapes and item categorizations. This way, when we add an urgent scrape to the front of the queue, it can be completed sooner.